### PR TITLE
prepare jenkinsfile and dockers to be deployed

### PR DESCRIPTION
**Pull request by houtmanj**

### CODE REVIEW

File: ci/Dockerfile.graphql [hunk 1]
```diff
@@ -18,5 +15,6 @@ FROM base
 ARG NODE_ENV
 RUN npm ci --no-color -q
 COPY apps/graphql/ .
-ENV PORT=80
+USER node
+ENV PORT=8080
```
Comment (afjlambert) [inline ci/Dockerfile.graphql:19]: Why not use port 80? Just curious.

File: ci/nginx-deployed.conf [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
     location /graphql {
-        proxy_pass http://backend/graphql;
+        set $backend "$BACKEND_URI";
+        proxy_pass $backend;
     }
 
     location / {
-        proxy_cache my_cache;
-        add_header X-Cache-Status $upstream_cache_status;
```
Comment (afjlambert) [inline ci/nginx-deployed.conf:21]: Why did you remove this part?

File: Jenkinsfile [hunk 3]
```diff
@@ -1,39 +1,127 @@
...
+    catch (Throwable t) {
+        sendMessage "Build failed", "failed", "danger"
+        throw t;
+    }
+    finally {
+        sendMessage "Build succeeded", "success"
+        if (tearDown) {
+            tearDown();
+        }
+    }
```
Comment (afjlambert) [inline Jenkinsfile:36]: Why did you refactor this?
It now reports build succeeded even if it failed.
Also `tearDown` is never used, so why include it?

File: Jenkinsfile [hunk 4]
```diff
@@ -1,39 +1,127 @@
 #!groovy
+// Project Settings for Deployment
+String PROJECTNAME = "vergunningcheck"
+String CONTAINERDIR = "."
+String PRODUCTION_BRANCH = "master"
+String ACCEPTANCE_BRANCH = "development"
```
Comment (afjlambert) [inline Jenkinsfile:6]: kun je deze aanpassen naar `chappie1.0`

File: ci/Dockerfile.graphql (hunk 1 above)
Comment (houtmanj) [reply ci/Dockerfile.graphql:19]: The node user is not allowed to bind to ports below 1024. 

Raising the port allows us to avoid running as root

File: Jenkinsfile (hunk 3 above)
Comment (houtmanj) [reply Jenkinsfile:36]: This was a copy paste from an example Jenkinsfile with the send messages added to keep that functionality in tact

Just say you would like to see it.

File: Jenkinsfile (hunk 4 above)
Comment (houtmanj) [reply Jenkinsfile:6]: you have this app in its own repo now;  
why not use the branch pattern that everyone uses?

File: Jenkinsfile (hunk 3 above)
Comment (afjlambert) [reply Jenkinsfile:36]: Please revert to the way it was. We'll improve on it later if needed.

File: Jenkinsfile (hunk 4 above)
Comment (afjlambert) [reply Jenkinsfile:6]: ok, I'll rename chappie1.0 to development and create master based on that.

**Comment by afjlambert** [comment]
Can you resolve the merge conflict too?
